### ISSUE TITLE
Release Artifacts using New Tag Format?

### ISSUE THREAD

**Issue by driverkt** [ProtonMail/inbox-desktop#36]
I use homebrew to install/update proton-mail's desktop app. See the cask formula [here](https://github.com/Homebrew/homebrew-cask/blob/7410c5370500ba19cfebf2723d1e24885d61778c/Casks/p/proton-mail.rb).

For the most recent release, it seems two 1.0.5 tags were created, namely `1.0.5` and `v1.0.5`. *This time* the platform-specific build artifacts were released under the `1.0.5` tag, rather than `v1.0.5`. When 1.0.5 was released, it looks like the maintainer of the homebrew cask formula [updated the version number and the digest for the downloads, but didn't modify the link](https://github.com/Homebrew/homebrew-cask/commit/a18f96c8de786c2d2cf9c416a34c3170a7680e9d), so the link to the file has the tag format prefixed with `v`. Will the tag format be changing? Or is it possible that the artifacts will be posted under a `v1.0.x` tag once again in the future?  

Basically, as I understand it, homebrew likes to have fairly stable url formats and just change the version number and digest for each release. 

See also: 

```
==> Upgrading 1 outdated package:
proton-mail 1.0.4 -> 1.0.5
==> Upgrading proton-mail
==> Downloading https://github.com/ProtonMail/inbox-desktop/releases/download/v1.0.5/Proton.Mail-1.0
curl: (22) The requested URL returned error: 404                                                    

==> Purging files for version 1.0.5 of Cask proton-mail
Error: proton-mail: Download failed on Cask 'proton-mail' with message: Download failed: https://github.com/ProtonMail/inbox-desktop/releases/download/v1.0.5/Proton.Mail-1.0.5-arm64.dmg
```
